src/plugins/home/public/application/components/add_data/add_data.tsx

```diff
@@ -66,19 +67,38 @@ export const AddData: FC<Props> = ({ addBasePath, application, isDarkMode }) =>
 
             <EuiSpacer />
 
-            <EuiFlexGroup gutterSize="m" responsive={false} wrap>
+            <EuiFlexGroup gutterSize="m">
+              {isCloudEnabled && (
+                <EuiFlexItem grow={false}>
+                  {/* eslint-disable-next-line @elastic/eui/href-or-on-click */}
+                  <EuiButton
+                    data-test-subj="guidedOnboardingLink"
+                    fill
+                    href={addBasePath('#/getting_started')}
+                    onClick={(event: MouseEvent) => {
+                      trackUiMetric(METRIC_TYPE.CLICK, 'guided_onboarding_link');
+                    }}
+                  >
+                    <FormattedMessage
+                      id="home.addData.guidedOnboardingLinkLabel"
+                      defaultMessage="Launch Guided setup"
```

Comment:
I'll defer to @kellyemurphy, but I think it makes sense for the text to match the button label in the header, so it would be something like: "Launch Setup guide"

Reply:
Yes, this might have been an older wording, let's see what @kellyemurphy says

Reply:
@alisonelizabeth is right on the money here.

Reply:
Thank you, @kellyemurphy! I updated the button label and used sentence case for it "Launch setup guide". Is that correct or should I change to "Launch Setup guide"?